hypershift-operator/controllers/scheduler/dedicated_request_serving_nodes.go

```diff
@@ -617,6 +617,11 @@ func (r *DedicatedServingComponentSchedulerAndSizer) updateHostedCluster(ctx con
 		hc.Annotations[hyperv1.APICriticalPriorityClass] = *sizeConfig.Effects.APICriticalPriorityClassName
 	}
 
+	resourceRequestAnnotations := resourceRequestsToOverrideAnnotations(sizeConfig.Effects.ResourceRequests)
+	for k, v := range resourceRequestAnnotations {
+		hc.Annotations[k] = v
+	}
```

Comment:
for my understanding, why are we using annotations? can't the CPO read the clustersizing CR directly?

Reply:
thanks.